package.json

```diff
@@ -14,9 +14,11 @@
     "postinstall": "touch secrets.js",
     "seed": "node script/seed.js",
     "start": "node server",
-    "start-dev": "NODE_ENV='development' npm run build-client-watch & npm run start-server",
+    "start-dev":
+      "NODE_ENV='development' npm run build-client-watch & npm run start-server",
```

Comment:
Nit: Why is there a line break here? Formatting? Probably best to keep it consistent with other package.json files… the long lines are a bummer, but we still get line breaks in narrow-enough editors with these lines even with the line breaks.

Reply:
I think my editor may have auto formatted it that way, I'll make a quick commit to bring them in line.